pkg/files/file.go

```diff
@@ -47,6 +47,12 @@ func NewSortedFilesFromPaths(paths []string, opts SymlinkAllowOpts) ([]*File, er
 
 		relativePath := ""
 		pathPieces := strings.Split(path, "=")
+		if strings.HasPrefix(pathPieces[0], "http://") || strings.HasPrefix(pathPieces[0], "https://") {
+			pathPieces = []string{path}
+		} else if len(pathPieces) > 1 && (strings.HasPrefix(pathPieces[1], "http://") || strings.HasPrefix(pathPieces[1], "https://")) {
```

Comment:
What is the scenario where this if statement is triggered?

Reply:
@joaopapereira this is a safetynet for the use case that the  file's path is overwritten on the command line; cf. the `case 2` part in L60.

Example: want to cover both scenarios of
- `path = "myOverwrittenPath=https://example.com?hello=world"`
- `path = "https://example.com?hello=world"`